src/app/shared/domain/definition/money-gbp-case-field.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Orderable } from '../../../core/order/orderable.model';
+import { FieldType } from './field-type.model';
+import { WizardPageField } from '../wizard-page-field.model';
+import { CaseField } from './case-field.model';
+
+export class MoneyGBPCaseField extends CaseField {
```

Comment:
shouldn't we call this new field PaymentTotal? 
This name is confusing as someone can think this is the field which represent monetary values

Reply:
This may be reused in that way in multiple places.

Reply:
Fix it.